### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Amrit131**
### Hello Microverse Project: Feature Branch

In this branch we have added-:

- Added a Readme file.
- Added a html file with a header saying Hello Microverse!
- A Stylesheet which changes the color of the header.
- A test.md file which is ignored using .gitignore.
- Added Linters for Html and CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<<<<<<< HEAD
+# hello-microverse
+This is a repository for my first project in Microverse program...
+=======
```
Comment (awais-amjed) [inline README.md:4]: Kindly remove these lines and make sure that the Readme resembles the [Microverse Readme Template](https://github.com/microverseinc/readme-template) as closely as possible, and only make the necessary changes 🤗

File: text.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (awais-amjed) [inline text.md:1]: Kindly add this file to `.gitignore` following the guidelines in your [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) 🤗

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Amrit131 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Descriptive commit messages ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [x] Kindly update the Pull Request description so that it has a short summary of the changes you made in this PR as described [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) 🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Amrit131) [reply README.md]: I will surely do the changes, Thank you! for the review.

**Comment by Amrit131** [comment]
> # STATUS: CHANGES REQUIRED ♻️
> Hi @Amrit131 ,
> 
> Good job so far! 👏👏 There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗
> 
> ## Highlights
> * [x]  Descriptive Pull Request ✔️
> * [x]  Descriptive commit messages ✔️
> * [x]  Good code organization ✔️
> 
> gif [ gif ](https://camo.githubusercontent.com/0b6134a224332efd68cb15c3a3ff40ac036be75801f5ab22f0905d6b343c8a60/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f714c57644d5958314e594632672f67697068792d646f776e73697a65642d6c617267652e676966) [ ](https://camo.githubusercontent.com/0b6134a224332efd68cb15c3a3ff40ac036be75801f5ab22f0905d6b343c8a60/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f714c57644d5958314e594632672f67697068792d646f776e73697a65642d6c617267652e676966)
> 
> ## Required Changes ♻️
> * [x]  Kindly update the Pull Request description so that it has a short summary of the changes you made in this PR as described [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) 🤗
> 
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amrit131** [comment]
Sorry for the late reply and replying through email, it's just that it's my
first code review and I am a little bit confused. So I resolved the issue
and clicked on the resolve issue button present in the branch but I don't
know if I need to open another code review request or just wait for you.

Thank you
Amrit

**Comment by Amrit131** [comment]
@awais-amjed So I just read that I need to tag you sorry for all this confusion.

**Comment by awais-amjed** [comment]
@Amrit131 , So after you are done making the changes mentioned in the Code Review, you should ask for another Code Review from your dashboard and someone will review your PR. 
Also you should not close or delete this branch or PR and after making the required changes in this branch, you should ask for another Code Review using this Same Pull Request as you did before.
This way the next code reviewer will have an idea of my Code Review and they will be able to check your code accordingly.
All of this should be mentioned in the Code Review process at the bottom of your project requirements.

**Comment by Amrit131** [comment]
@awais-amjed, Thank you for helping me out, I will make sure to follow the instructions. Also about closing the pull request/branch it was a mistake from my side, in panic I got confused a little bit and closed the PR/branch while trying to do something else. I will make sure it never happens again.
Thank you.

**Review by sumairq — APPROVED** [review]
### Hi @Amrit131 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amrit131** [comment]
Thank you! Sumair.

**Pull request merged**